### ISSUE TITLE
Need to have each account with id

### ISSUE THREAD

**Issue by imougy** [passiv/snaptrade#3]
It will be great to add an **id** entry to the account as well as **favicon**

**Comment by nickmccullum** [comment]
Which endpoint, @imougy ?

**Comment by imougy** [comment]
Hi @nickmccullum , while calling holdings [API](https://app.swaggerhub.com/apis/passiv/PassivSnapAPI/1.0.0#/Account%20Details/get_snapTrade_holdings), this is the return:-
```
[
  {
    "account": {
      "brokerage": "Passiv Investr",
      "number": "P1334567",
      "name": "Registered Retirement Savings Account"
    },
```
would be great to add id and favicon

**Comment by nickmccullum** [comment]
@imougy you mean the favicon of the brokerage, right?

**Comment by imougy** [comment]
@nickmccullum Correct.  Here is a plist file I use to display things (had to zip it for attachment purpose).
<img width="591" alt="Screen Shot 2022-01-13 at 11 17 26 AM" src="https://user-images.githubusercontent.com/3361440/149394914-98515e40-f8b8-49ac-8c86-f6b0b1e1db25.png">

[passiv.plist.zip](https://github.com/passiv/snaptrade/files/7865387/passiv.plist.zip)
